src/index.ts

```diff
@@ -73,13 +71,6 @@ const variableinspector: JupyterFrontEndPlugin<IVariableInspectorManager> = {
       return panel;
     }
 
-    // Enable state restoration
-    restorer.restore(tracker, {
-      command,
-      args: () => ({}),
-      name: () => 'variableinspector'
-    });
```

Comment:
Does this means that variable inspector sidebar will no longer re-open on refresh if it was the active one?

Is this required?

Reply:
Ah sorry I will revert this, this is an leftover from me debugging